### PR TITLE
feat: globals option

### LINKED ISSUE #30: A new option to set `globals`
When using externals with UMD builds, rollup tries to guess the global name of the module and throws a warning that it may be specified with `globals` option.

### PR DESCRIPTION
**Pull request by pi0**
resolves #30 

Tests and Docs also included

### CODE REVIEW

File: src/get-rollup-options.js [hunk 1]
```diff
@@ -202,6 +202,21 @@ export default function(options, format) {
     external = [external]
   }
 
+  let globals
+  if (options.globals) {
+    globals = options.globals
+    if (typeof globals === 'string') {
+      globals = {}
+      options.globals.split(',').forEach(g => {
+        const s = g.split(':')
```
Comment (egoist) [inline src/get-rollup-options.js]: maybe add note about such usage to the doc as well?

Comment (pi0) [reply src/get-rollup-options.js]: It is already added to cli usage. But i can add it to docs too :)

Comment (egoist) [reply src/get-rollup-options.js]: I'm not sure if it's necessary though, you can in fact use sth like `--global.foo foo --global.bar bar`

**Comment by pi0** [comment]
Sorry, i just forgot about this PR. Really thanks @egoist for completion of changes ❤️

**Pull request merged**